samples/cartridges/openstack/wso2as-521/wso2as-manager.json

```diff
@@ -0,0 +1,60 @@
+{
+  "type": "wso2as-521-manager",
+  "category": "framework",
+  "provider": "wso2",
+  "host": "as.wso2.org",
+  "displayName": "WSO2 AS 5.2.1 Manager",
+  "description": "WSO2 AS 5.2.1 Manager Cartridge",
+  "version": "5.2.1",
+  "multiTenant": false,
+  "loadBalancingIPType": "public",
+  "portMapping": [
+    {
+      "name": "mgt-http",
+      "protocol": "http",
+      "port": 9763,
+      "proxyPort": 0
+    },
+    {
+      "name": "mgt-https",
+      "protocol": "https",
+      "port": 9443,
+      "proxyPort": 0
+    }
+  ],
+  "iaasProvider": [
+    {
+      "type": "openstack",
+      "imageId": "RegionOne/abab3e25-0d53-444b-b7f4-d7c9c21b9bf0",
+      "property": [
+        {
+          "name": "instanceType",
+          "value": "RegionOne/15c3065c-462c-4977-9143-094d63d1c2c7"
+        },
+        {
+          "name": "keyPair",
+          "value": "anuruddha"
+        },
+        {
+          "name": "securityGroups",
+          "value": "default"
+        }
+      ],
+      "networkInterfaces": [
+        {
+          "networkUuid": "512e1f54-1e85-4dac-b2e6-f0b30fc552cf"
+        }
+      ]
+    }
```

Comment:
Above OpenStack values would not work in any environment, may be they are specific to your environment. We might need to either keep them empty or add a place holder text.